### PR TITLE
#33 Reponsive for mobile

### PR DESCRIPTION
**Pull request by trinhlaosu**
Issue #33 Reponsive for mobile

### CODE REVIEW

File: dmd-frontend/src/components/App.tsx [hunk 1]
```diff
@@ -46,7 +46,7 @@ const App = () => {
       <Topbar />
       <Container className={classes.container}>
         <Editor className={classes.editor} onSaveFunc={onEditorSave}></Editor>
-        <div className={classes.sidebar}></div>
+        {/* <div className={classes.sidebar}></div> */}
```
Comment (chiro-hiro) [inline dmd-frontend/src/components/App.tsx:49]: Do not commit commented code

Comment (chiro-hiro) [reply dmd-frontend/src/components/App.tsx]: @trinhlaosu Please response, I won't merge the code without resolved conversation.

Comment (trinhlaosu) [reply dmd-frontend/src/components/App.tsx]: I'm sorry.
I removed that commented code in source.

**Comment by chiro-hiro** [comment]
@trinhlaosu Please link issue add to project and milestone as well.

**Pull request merged**